python/nemo_relay/llm.py

```
@@ -404,7 +414,7 @@ def request_intercepts(name, request):
     return _native_llm_request_intercepts(name, request)
 
 
-def conditional_execution(request):
+def conditional_execution(request: LLMRequest) -> None | Awaitable[None]:
```

Ruff (0.16.2)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)